ui/components/src/ui-components/table/component/Table.scss

```diff
@@ -131,6 +131,7 @@
 
   .react-grid-Cell__value {
     // Remove transformations that center the content vertically in order to make cell borders fit
+    overflow: visible !important;

```

Comment:
You inserted this new style between a comment and the style that it's commenting. Please move the new style up/down and add a comment explaining why the override is needed.

Reply:
Fixed